webapp/components/interop-dashboard.js

```diff
@@ -672,5 +708,74 @@ class InteropDashboard extends PolymerElement {
     this.$.toggleStable.setAttribute('aria-pressed', true);
     this.$.toggleExperimental.setAttribute('aria-pressed', false);
   }
+
+  getSortIcon(index) {
+    if (this.toggleStates[index]) {
+      return '/static/expand_more.svg';
+    }
+    return '/static/expand_less.svg';
+  }
+
+  sortRows(rows, index, tableState) {
```

Comment:
This method is pretty complex and is sorting two different ways based on whether you're sorting alphabetically or by score. Maybe each of the sorts can be broken into separate functions?